assets/js/common/forms.js

```diff
@@ -145,6 +146,71 @@ function handleConditionalRadios() {
     });
 }
 
+// Track occurrences of users changing their mind about radio options
+// eg. to highlight potential confusion around questions
+function trackIndecisiveness() {
+    let fields = {};
+    $('input[type="radio"]').on('click', function() {
```

Comment:
General comment on all of these listeners: It would be good to restrict these selectors to the main form (i.e. be certain we're not listening to other forms).

Reply:
Done!